### PR TITLE
added documentation for OTRS5 about procmail https://portal.otrs.com/…

### PR DESCRIPTION
**Pull request by TobiasSommer**
added from https://portal.otrs.com/otrs/customer.pl?Action=CustomerFAQZoom;ItemID=34;Nav=

### CODE REVIEW

File: en/content/administration/email/email.xml [hunk 1]
```diff
@@ -112,6 +112,32 @@ linux:/opt/otrs/bin#
...
+
+<title>Routing via procmail using otrs.Console.pl</title>
+
+<para>
+In order to route mails in a specific queue using otrs.Console.pl use the following example.
+</para>
+
+<para>
+<screen>
+| $SYS_HOME/bin/otrs.Console.pl Maint::PostMaster::Read --target-queue=QUEUENAME
```
Comment (mgruner) [inline en/content/administration/email/email.xml]: Here my recommendation would be to wrap the content in the screen tag in a <!CDATA[[ ... ]]> block. This avoids problems with < and > characters inside the contend. We do that in some places already, not everywhere.

Comment (mgruner) [inline en/content/administration/email/email.xml]: I would always do that for screen and programlisting tags.

**Comment by mgruner** [comment]
Cool, looks very nice in general! I'll wait for a small update from you.

**Comment by dignin** [comment]
You mean like

<programlisting language="Perl"><![CDATA[
    # Job Name: ExternalNumber
    # External Ticket Number Reconition, check for Incident-<number> in incoming mails subject and
    # body from the addeesses <sender>@externalticket.com, if number is found it will be stored in
    # the dynamic field 'ExternalNumber' (that need to be setup in the Admin Panel).
    $Self->{'PostMaster::PreFilterModule'}->{'000-ExternalTicketNumberRecognition5'} =  {
        'FromAddressRegExp' => '\\s*@externalticket.com',
        'NumberRegExp'      => 'Incident-(\\d.*)',
        'SearchInSubject'   => '1',
        'SearchInBody'      => '1',
        'TicketStateTypes'  => 'new;open'
        'DynamicFieldName'  => 'ExternalNumber',
        'Module'            => 'Kernel::System::PostMaster::Filter::ExternalTicketNumberRecognition',
        'Name'              => 'Test External Ticket Number',
        'SenderType'        => 'system',
    };
    ]]></programlisting>

**Comment by mgruner** [comment]
Exactly. https://de.wikipedia.org/wiki/CDATA

**Comment by mgruner** [comment]
Merged in 3fd68932163c70191272ec815fd29d5a89ce4a2e. Thanks a lot, @TobiasSommer and @dignin! MORE!

**Pull request closed without merging**